src/teams.js

```diff
@@ -1,58 +1,52 @@
-import { isMemberPropValid, createMemberPatchQuery } from './utils'
-
-export default http => {
-  return {
-    get: () => {
-      return getTeam(http)
-    },
-    addMembers: args => {
-      return addMembers(http, args)
-    },
-    removeMembers: args => {
-      return removeMembers(http, args)
-    }
+import { isMemberPropValid, createMemberPatchQuery } from './utils';
+
+export default http => new Teams(http);
+
+class Teams {
+  constructor(_http) {
+    this._http = _http;
   }
-}
-
-const getTeam = http => {
-  return http.request({
-    method: 'get',
-    url: '/teams/mine'
-  })
-}
-
-const addMembers = (http, { members }) => {
-  if (!isMemberPropValid(members)) {
-    throw `No member provided`
+
+  get() {
```

Comment:
Is this public? If we are changing not internal but API method names the commits should be tagged as breaking changes.

Reply:
I have the feeling that just the word _get_ isn't explicit enough, we should have something clearer around what we are returning here in my opinion.

Reply:
@jeremenichelli It is public => https://developer.typeform.com/create/teams/#walkthroughs

While we could offer a more explicit name (`getMine`?) I didn't want to necessarily cause any breaking changes, unless we renamed the function and then use `get` as a pass through to the renamed function for now.

Reply:
Well, if it's public and we change its name it is a breaking change. It's fine, but you need to label the commit as `BREAKING CHANGE` in the footer of the commit message so the version bump is correct. about the name, I think that `getMine` is good.